config/install/bento_sdk.settings.yml

```
@@ -27,4 +27,23 @@ retry:
+    order_paid: true
+    order_fulfilled: true
+    order_cancelled: true
+    cart_abandoned: true
+  cart_abandonment:
+    threshold_hours: 24
+    check_interval: 60
+  data_enrichment:
+    include_product_details: true
+    include_customer_context: true
+    include_order_context: true
+    include_product_images: false
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)